examples/gallery/maps/country_polygons.py

```diff
@@ -0,0 +1,103 @@
+"""
+Highlight country and continent polygons
+----------------------------------------
+The :meth:`pygmt.Figure.coast` method can
+highlight country polygons via the ``dcw``
+parameter. It accepts the country code or
+full country name and can draw its borders
+and add a color to its landmass. It's also
+possible to define multiple countries at
+once by separating the indiviudal names
+with commas.
+"""
+
+# sphinx_gallery_thumbnail_number = 2
+
+
+import pygmt
+
+fig = pygmt.Figure()
+
+fig.basemap(
+    region=[-12, 32, 34, 62],
```

Comment:
Adjust study region corresponding to Scandinavia:
```suggestion
    region=[-12, 32, 34, 72],
```